packages/core/components/StatusMessage/index.tsx

```diff
@@ -45,31 +47,50 @@ export default function StatusMessage() {
     const dispatch = useDispatch();
 
     return (
-        <Stack {...verticalStackProps} className={styles.container}>
+        <Stack {...verticalStackProps} verticalAlign="end" className={styles.container}>
             {map(
                 useSelector(interaction.selectors.getProcessStatuses),
                 (statusUpdate: StatusUpdate) => {
                     const {
                         data: { msg, status = ProcessStatus.NOT_SET, progress },
                         onCancel,
                     } = statusUpdate;
+                    let onDismiss; // If has cancel option, don't show dismiss button
                     let cancelButton;
                     if (onCancel) {
-                        cancelButton = <DefaultButton onClick={onCancel}>Cancel</DefaultButton>;
-                    }
+                        cancelButton = (
+                            <SecondaryButton
+                                iconName=""
+                                title="Cancel"
+                                text="CANCEL"
+                                onClick={onCancel}
+                            />
+                        );
+                    } else
+                        onDismiss = () =>
+                            dispatch(interaction.actions.removeStatus(statusUpdate.processId));
 
                     return (
                         <MessageBar
                             className={classNames(styles.messageBar, {
                                 [styles.success]: status === ProcessStatus.SUCCEEDED,
-                                [styles.error]: status === ProcessStatus.FAILED,
+                                [styles.warning]: status === ProcessStatus.WARNING,
+                                [styles.error]:
+                                    status === ProcessStatus.FAILED ||
+                                    status === ProcessStatus.ERROR,
                             })}
                             actions={cancelButton}
                             key={statusUpdate.processId}
                             messageBarType={statusToTypeMap[status]}
-                            onDismiss={() =>
-                                dispatch(interaction.actions.removeStatus(statusUpdate.processId))
-                            }
+                            styles={{
+                                iconContainer:
+                                    status === ProcessStatus.STARTED
```

Comment:
Not sure if this is the line that causes this, but I think users should be able to dismiss messages about files that are downloading. If a user tries to download 20 images they might get blocked by all the messages.

Reply:
Good point. One issue is that the download process currently resends the status message as progress continues, so dismissing once isn't permanent while it's ongoing. Maybe we could have a way of minimizing it/making the messages more compact instead of dismissing fully?

Reply:
Huh... that is annoying. I guess if it isn't a new blocker we are creating and it is just an existing potential blocker we aren't fixing then ignoring it for this PR is fine.